src/vehicle.cpp

```diff
@@ -5291,6 +5294,15 @@ void vehicle::refresh()
     rail_wheel_bounding_box.p1 = point( railwheel_xmin, railwheel_ymin );
     rail_wheel_bounding_box.p2 = point( railwheel_xmax, railwheel_ymax );
 
+    if( !refresh_done ) {
+        mount_min.x = 0;
```

Comment:
You won't need this three rows if you assign everythinh in the fourth one.

Reply:
Yes, already fixed, thanks!